packages/admin-ui/src/pages/Customers/Customers.jsx

```diff
@@ -120,6 +111,8 @@ const Customers = () => {
 
   const byAuthorized = c => (c.authorizedStatus.label === 'Pending' ? 0 : 1)
   const byLastActive = c => new Date(R.prop('lastActive', c) ?? '0')
+
+  const customers = R.path(['customers'])(customersResponse) ?? []
```

Comment:
`?? []` is unnecessary in this statement or the following.

Reply:
One of the two has to do it. R.map is not null safe.